modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -21,65 +21,57 @@ To configure forwarding application logs from a project, you must create a `Clus
 .Example `ClusterLogForwarder` CR
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance <1>
-  namespace: openshift-logging <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
+  managementState: Managed
   outputs:
-   - name: fluentd-server-secure <3>
-     type: fluentdForward <4>
-     url: 'tls://fluentdserver.security.example.com:24224' <5>
-     secret: <6>
-        name: fluentd-secret
-   - name: fluentd-server-insecure
-     type: fluentdForward
-     url: 'tcp://fluentdserver.home.example.com:24224'
-  inputs: <7>
+   - name: <output_name> #<1>
+     type: <output_type> #<2>
+     #...
+  inputs: #<3>
    - name: my-app-logs
      application:
         namespaces:
-        - my-project <8>
+        - my-project #<4>
```

Comment:
This needs include/exclude and namespace/container.     see note above, and the example in our upstream (or use `oc explain clf.spec.inputs.application`)